src/components/ReviewCycles/ReviewCycleModals/useReviewCycleCreateModal.tsx

```diff
@@ -0,0 +1,37 @@
+import useModal from 'hooks/useModal';
+import { Headline4 } from 'styles/typography';
+import ReviewCycleForm from '../ReviewCycleForm';
+import { message } from 'antd';
+import { useCreateReviewCycle, useGetReviewCycles } from '@apis/review/useReviewCycles';
+
+function useReviewCycleCreateModal() {
+  const modal = useModal();
+
+  const { refetchReviewCycle } = useGetReviewCycles();
+  const { createReviewCycle } = useCreateReviewCycle({
+    onSuccess: () => {
+      message.success('새로운 리뷰 정책을 생성했어요.');
+      modal.close();
+      refetchReviewCycle();
+    },
+    onError: message.error,
+  });
+
+  function render() {
+    return modal.render({
+      title: <Headline4>새로운 리뷰 정책 만들기</Headline4>,
+      visible: modal.visible,
+      children: (
+        <ReviewCycleForm onFinish={createReviewCycle} onReset={modal.close} confirmButtonProps={{ text: '생성하기' }} />
+      ),
+    });
+  }
+
+  return {
+    render,
+    open: modal.open,
+    close: modal.close,
+  };
```

Comment:
요런 식으로 기존 `modal`을 확장하는 훅으로 만들어도 좋을 것 같아요~
```suggestion
  return {
    ...modal,
    render,
  };
```